tests/test_application.py

```diff
@@ -1680,3 +1680,28 @@ async def raise_method(*args, **kwargs):
                 app.run_webhook(close_loop=False, stop_signals=None)
 
         assert len(recwarn) == 0
+
+    @pytest.mark.dev
+    def test_signal_handlers(self, app, monkeypatch):
+        # this test should make sure that signal handlers are set by default on Linux + Mac,
+        # and not on Windows.
+
+        def signal_handler_test(*args, **kwargs):
+            # args[0] is the signal, [1] the callback
+            assert args[0] in (signal.SIGINT, signal.SIGTERM, signal.SIGABRT)
+
+        loop = asyncio.get_event_loop()
+        monkeypatch.setattr(loop, "add_signal_handler", signal_handler_test)
+
+        async def abort_app(context):
+            raise KeyboardInterrupt
```

Comment:
nice 👍 then we can maybe even refactor the other tests to also run on windows with this trick :)